### PR TITLE
 Creates `Modal` component

### PR DESCRIPTION
**Pull request by schalkventer**

### CODE REVIEW

**Comment by schalkventer** [comment]
image

**Comment by schalkventer** [comment]
image

**Comment by schalkventer** [comment]
@jbothma Added link. Let me know if you approve? :)

File: _includes/components/homepage/Expenditure/index.jsx [hunk 1]
```diff
@@ -9,7 +9,7 @@ function Expenditure() {
     for (let i = 0; i < componentsList.length; i++) {
       const component = componentsList[i];
       const rawItems = JSON.parse(component.getAttribute('data-info')).data;
-      const year = component.getAttribute('data-year') || '2017-18';
```
Comment (jbothma) [inline _includes/components/homepage/Expenditure/index.jsx]: did you try a couple of years to see that this still works?

File: _includes/components/universal/Graph/index.jsx [hunk 2]
```diff
@@ -9,7 +9,7 @@ for (let i = 0; i < componentList.length; i++) {
   const { data } = JSON.parse(component.getAttribute('data-graph'));
   const legendAttribute = component.getAttribute('data-config') || '{ "legend": [] }';
   const { legend } = JSON.parse(legendAttribute);
-  const year = component.getAttribute('data-year') || '2017-18';
```
Comment (jbothma) [inline _includes/components/universal/Graph/index.jsx]: did you try a couple of years to see that this still works?

**Comment by schalkventer** [comment]
@jbothma Yeah, switch around between years on a couple of pages.

**Pull request merged**